### PR TITLE
Gandi Packet Journey support.

### PR DESCRIPTION
**Pull request by xbeaudouin**
Hi there,

This is a pull request to add Gandi Packet Journey support to Librenms.

Since this is a "raw linux" with DPDK and some Gandi's software, available on https://github.com/Gandi/packet-journey, I have just modified the linux classes and added one "brand" of linux.

Regards

### CODE REVIEW

**Comment by laf** [comment]
Thanks.

Codewise looks fine. Had to check that the $os value is the right one but they seem to refer to it themselves as that so all good.

Can you confirm the source of the logo and mib please?

**Comment by xbeaudouin** [comment]
Logo and MIB source are comming from my employer : Gandi :D

**Comment by laf** [comment]
lol well ok then :)

:+1:  from me.

@librenms/reviewers

File: includes/discovery/os/linux.inc.php [hunk 1]
```diff
@@ -50,6 +50,20 @@
                 $os = 'airos-af';
             }
         }
+        else if (trim(snmp_get($device, 'GANDI-MIB::rxCounter.0', '-Osqnv', 'GANDI-MIB')) != '') {
```
Comment (murrant) [inline includes/discovery/os/linux.inc.php]: I have been preferring:
`else if (snmp_get($device, 'GANDI-MIB::rxCounter.0', '-Osqnv', 'GANDI-MIB') !== false) {`

Other than that minor preference :+1:

**Comment by xbeaudouin** [comment]
Well I can do this on monday (the repos I am working on is on my office computer) ?

File: includes/discovery/os/linux.inc.php (hunk 1 above)
Comment (xbeaudouin) [inline includes/discovery/os/linux.inc.php]: Well I can do this on monday - My repo is in my office :D

**Comment by xbeaudouin** [comment]
Update done as suggested @murrant :D

**Pull request merged**